app/models/manageiq/providers/autosde/storage_manager/event_catcher/runner.rb

```diff
@@ -25,10 +25,21 @@ def monitor_events
 
   def queue_event(event)
     _log.info("#{log_prefix} Caught event AutoSDE [#{event.event_id}]")
+    add_notification_type(event)
     event_hash = event_to_hash(event, @cfg[:ems_id])
     EmsEvent.add_queue('add', @cfg[:ems_id], event_hash)
   end
 
+  def add_notification_type(event)
+    NotificationType.create_with(
+      :name       => event.event_type,
+      :level      => "warning",
+      :audience   => "global",
+      :expires_in => 7.days,
+      :message    => "%{event_type} in event %{uuid}, error code: %{error_code}. %{description}"
+    ).find_or_create_by(:name => event.event_type)
+  end
+
```

Comment:
@Lauferin notification_types are seeded on startup so you don't have to create these on the fly.  They are seeded from https://github.com/ManageIQ/manageiq/blob/fd400ab58f82496f641ffa3f3147fc12e38345b7/db/fixtures/notification_types.yml

I would like to make this pluggable so you can define these in this repo but for now for testing you can add your event types you want notifications for to that yaml file.

Reply:
I've already tested all what was needed... so I guess we freeze this PR until the pluggable is done?